docs/topics/manage/limits.md

```diff
@@ -18,7 +18,7 @@ Momento Topics seek to protect themselves and their customers when it comes to s
 
 :::info
 
-Since Topics utilizes Momento Cache, so all [Cache limits](/cache/manage/limits) apply to Topics as well.
+Since Topics utilizes Momento Cache, all [Cache limits](/cache/manage/limits) apply to Topics as well.
```

Comment:
On line 17 above (I can't comment on it) should we update the language to `10 public operations/sec`? Not sure it matters, was just thinking for consistency.